src/native/event_loop_group.c

```diff
@@ -55,21 +67,57 @@ jlong JNICALL
     return (jlong)elg;
 }
 
+struct event_loop_group_cleanup_callback_data {
+    JavaVM *jvm;
+    jlong elg_addr;
+    jobject java_event_loop_group;
+};
+
+static void s_event_loop_group_cleanup_completion_callback(void *user_data) {
+    struct event_loop_group_cleanup_callback_data *callback_data = user_data;
+
+    // Elg memory can be freed
+    struct aws_event_loop_group *elg = (struct aws_event_loop_group *)callback_data->elg_addr;
+    aws_mem_release(elg->allocator, elg);
+
+    AWS_LOGF_DEBUG(AWS_LS_IO_EVENT_LOOP, "Event Loop Shutdown Complete");
```

Comment:
Should probably make a Java/JNI log subject instead, otherwise this might get confusing later.

Reply:
Yeah, I wasn't sure what to do with this so that was the placeholder and then I forgot about it